Sources/Workspace/WorkspaceState.swift

```diff
@@ -63,7 +63,8 @@ public final class WorkspaceState: SimplePersistanceProtocol {
             try self.persistence.restoreState(self)
         } catch {
             // FIXME: We should emit a warning here using the diagnostic engine.
-            print("\(error)")
+            TSCBasic.stderrStream.write("\(error)\n")
+            TSCBasic.stderrStream.flush()
```

Comment:
This doesn't hurt, but `stderr` is usually line-buffered (whereas `stdout` is usually fully buffered), so the second line here is most likely not needed.  But it's fine to leave it as a belt-and-suspenders kind of thing.

Reply:
That's right, there's buffering in TSC's `WritableByteStream`, though, which comes into play here. I was considering a more direct write but that's a couple lines of code so I preferred using the TSC type here.